docs/index.html

```diff
@@ -428,7 +428,7 @@ <h1 class="bg-white mb-6 xl:mb-12 text-3xl font-bold tracking-tight text-slate-9
   
   
   <li>
-    <a href="/article-example">Article Example</a>
+    <a href="/ai-twin">AI Twins: The New Challenge in Industrial Cyber-Physical Systems?</a>
```

Comment:
"AI Twins: The New Opportunity in Industrial Cyber-Physical Systems?" @teddynecsoiu can we go with this please?